Makefile

```diff
@@ -3,7 +3,7 @@ build:
 .PHONY: build
 
 build-image:
-	docker build .
+	podman build .
```

Comment:
docker build did not work on my local, got: 
```
Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container error was
```
Could be a network issue, but can roll it back to docker if that's whats in production